media/css/mozorg/home/home-2018.scss

```diff
@@ -7,14 +7,10 @@ $image-path: '/media/protocol/img';
 
 @import '../../../protocol/css/includes/lib';
 @import '../../../protocol/css/components/billboard';
-@import '../../../protocol/css/components/button';
-@import '../../../protocol/css/components/footer';
 @import '../../../protocol/css/components/modal';
 @import '../../../protocol/css/components/newsletter-form';
 @import '../../../protocol/css/templates/card-layout';
 
-@import '../../hubs/nav-download-button-remove';
```

Comment:
Note for reviewer: I decided to unhide the nav download button on the home page, since it now no longer conflicts with the positioning of the download button in the main banner. Happy to re-hide it again if there's a reason I'm not aware of.